spec/models/service_spec.rb

```diff
@@ -65,6 +66,100 @@
       end
     end
 
+    context 'with a user generated slug' do
+      describe 'with invalid slug' do
+        context 'when the slug contains capital letters' do
+          before do
+            subject.slug = 'aCaptialLetter'
+          end
+
+          it 'adds an error on slug' do
+            subject.valid?
+            expect(subject.errors[:slug]).to_not be_empty
```

Comment:
I prefer to explicitly test for the error message you are expecting, rather than just testing for any error message at all.  Firstly, it ensures you're not getting errors for another reason, and secondly, it makes the test much more understandable - specifying what you are expecting to be wrong.